.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: true
```

Comment:
what about adding here link to documentation and link to esp32 forum?

Reply:
links added